int8 level  # 0=最小, 1=中等, 2=最大
---
bool success